domain/src/main/kotlin/no/nav/su/se/bakover/domain/Behandling.kt

```diff
@@ -352,11 +351,13 @@ data class Behandling(
     ) :
         RuntimeException(msg)
 
-    sealed class IverksettFeil {
-        class AttestantOgSaksbehandlerErLik(val msg: String = "Attestant og saksbehandler kan ikke vare samme person!") :
-            IverksettFeil()
-
-        class Utbetaling(val msg: String) : IverksettFeil()
+    sealed class IverksettFeil(
+        open val msg: String = "Iverksett Feil"
+    ) {
+        data class AttestantOgSaksbehandlerErLik(override val msg: String = "Attestant og saksbehandler kan ikke vare samme person!") : IverksettFeil()
```

Comment:
Jeg er litt skeptisk til å legge på en meldingstekst i domenet hvis jeg ikke har bruk for det i forbindelse med logging/metrikker e.l. Og det har jeg sjeldent siden jeg sjeldent ønsker noe felles, men heller vil ha custom alt etter som hva jeg prøver å gjøre.

Reply:
Forsåvidt enig.